sdk/core/System.ClientModel/tests/client/ClientShared/ModelReaderWriterExtensions.cs

```diff
@@ -176,15 +176,16 @@ public static void WriteNumberValue(this Utf8JsonWriter writer, DateTimeOffset v
         writer.WriteNumberValue(value.ToUnixTimeSeconds());
     }
 
-    public static void WriteObjectValue(this Utf8JsonWriter writer, object? value)
+    public static void WriteObjectValue(this Utf8JsonWriter writer, object? value, ModelReaderWriterOptions? options = null)
```

Comment:
Curious: is this a change that will be needed in the generator to write the ModelReaderWriterExtensions file differently to support the proxy feature?